cdap-docs/admin-manual/source/installation/installation.rst

```diff
@@ -309,6 +309,24 @@ recommended installation is a minimum of two boxes.
 
 This will download and install the latest version of CDAP with all of its dependencies. 
 
+Direct Parcel Access
```

Comment:
This may be a little out of context on the installation page... should this just be on the cloudera manager integration page?  Parcels are only useful to Cloudera Manager users.  We should at least say something like "If you are using Cloudera Manager..."